apps/crn-server/src/routes/research-outputs.route.ts

```diff
@@ -27,11 +31,31 @@ export const researchOutputRouteFactory = (
   researchOutputRoutes.get(
     '/research-outputs',
     async (req, res: Response<ListResearchOutputResponse>) => {
-      const { query } = req;
+      const { query, loggedInUser } = req;
+
+      const { associationId, status, ...options } = query.status
+        ? validateResearchOutputDraftFetchOptions(query)
```

Comment:
ok so i think you want to have a single query string validation per endpoint, doing this conditionally seems very strange. If you know that some properties only refer to a situation when the drafts are being fetched - you can still apply that in ajv